lib/order.rb

```diff
@@ -0,0 +1,29 @@
+require_relative './menu.rb'
+
+class Order
+
+	attr_reader :order
+
+		@price_list = {
+		"fried eggs with bacon" => 6.00,
+		"eggs benedict with avocado" => 7.50,
+		"scrambled eggs with mozzarella" => 7.00,
+		"pancakes with chocolate" => 7.50,
+		"yoghurt with fresh fruit" => 6.00
+  }
+
+	@order = {}
+
+	def select_items(item, num_of_portions)
```

Comment:
I like how you've added number of portions available, that's something I didn't even think of.